### PR TITLE
fix ubuntu service script

### PR DESCRIPTION
**Pull request by fatihky**

### CODE REVIEW

File: daemon_linux_systemv.go [hunk 1]
```diff
@@ -235,23 +235,29 @@ lockfile="/var/lock/subsys/$proc"
...
         echo "$(date)" >> $stdoutlog
         $exec {{.Args}} >> $stdoutlog 2>> $stderrlog &
         echo $! > $pidfile
         touch $lockfile
-        success
+        echo "service $servname started"
         echo
     else
-        failure
+        # failure
```
Comment (fatihky) [inline daemon_linux_systemv.go]: I couldn't find where the `failure` is, so I simply commented it out.

File: daemon_linux_systemv.go [hunk 2]
```diff
@@ -235,23 +235,29 @@ lockfile="/var/lock/subsys/$proc"
...
+    fi
+
     if ! [ -f $pidfile ]; then
         printf "Starting $servname:\t"
         echo "$(date)" >> $stdoutlog
         $exec {{.Args}} >> $stdoutlog 2>> $stderrlog &
         echo $! > $pidfile
         touch $lockfile
-        success
+        echo "service $servname started"
```
Comment (fatihky) [inline daemon_linux_systemv.go]: I couldn't find where the `success` is, so I replaced it with simple message.

**Comment by takama** [comment]
Fatih, sorry for delay.
- The pull requests will be accepted only in "develop" branch
- Why you add the line 257 'echo "service $servname started"' - probably 'echo' without anything will do the same

Thank you for your patience and understanding

**Comment by fatihky** [comment]
I didn't know that. Can you delete that `echo` after merging pr? And, I will try to re-send pr to `develop` branch.

**Pull request closed without merging**